app/helpers/application_helper.rb

```diff
@@ -24,7 +24,11 @@ def dashboard_stylesheet_tag(filename)
   end
 
   def hot_javascript_tag(filename)
-    javascript_include_tag hot_javascript_path(filename)
+    js_path = hot_javascript_path(filename)
+    if (filename == 'campaigns') || (filename.include? 'survey')
```

Comment:
any way to avoid this?

Reply:
One way would to be to have do this all manually in all the haml files which need JQuery. Or I suppose I could include the JQuery from the JS itself. What do you think?

Reply:
I think including the jquery from the campaigns and surveys js files makes the most sense, just in terms of making sure the js files are the places that keep the details about what they require to work properly.